### PR TITLE
Adding explanation of stability and semver

### PR DESCRIPTION
**Pull request by arschles**
This patch moves the module stability explanations into a new document, and also adds explanation of how module stabilities move alongside semver releases

### CODE REVIEW

File: docs/roadmap.md [hunk 1]
```diff
@@ -15,9 +15,11 @@ See below for more information concerning stability plans for each.
...
 As services are developed, the plans available and features of the service may change and are not guaranteed to exist in the final stable release. When a service is promoted to stable, we will ensure backward compatibility. Services/Plans have three stability tiers:
 
-- `experimental` - We have an idea for a new service that we want to support, but don't have a good idea how it should look. We are essentially “throwing something against the wall” to see if it'll stick. Experimental services may be radically changed or removed at any time
-- `preview` - We have a better understanding how a service should look, but we don't yet guarantee backward compatibility. We do guarantee that this service won't go back to `experimental`, so we are committing to making it exist in some shape or form
-- `stable` - We now understand usage patterns of the service very well and we guarantee full backward compatibility
+- `experimental`
+- `preview`
+- `stable`
+
+Please see [stability documentation](./docs/stability.md) for information on each of these tiers.
```
Comment (carolynvs) [inline docs/roadmap.md]: I think this link should be `(stability.md)`. I got a 404 when trying to navigate otherwise.

Comment (arschles) [reply docs/roadmap.md]: thanks!!!

File: docs/stability.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+# Service Stability and OSBA Version
+
+We correlate service stability changes to OSBA version changes according to the
+following rules:
+
+* If a service stability goes from `experimental` to `preview`, a semver MINOR
+  or MAJOR release must happen
+* If a service stability goes from `preview` to `stable`, a semver MINOR
+  or MAJOR release must happen
+* If a service stability goes from `preview` to `experimental`, a semver
```
Comment (carolynvs) [inline docs/stability.md]: This contradicts the point below. Can we remove this point?

File: docs/stability.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+But when we increase a semver MAJOR, MINOR or PATCH version, we also take into
+account if any modules change in stability. Read on for how this works.
+
+# Service Stability
+
+We indicate a stability for each service in the OSBA catalog and docs to
+indicate how mature it is. Each service may have a different stability:
+
+* `experimental` - We have an idea for a new service that we want to support,
```
Comment (carolynvs) [inline docs/stability.md]: I'm confused by these definitions. I _thought_ that when we talk about service stability with regards to OSBA we are referring to the actual Azure Service's advertised stability. We aren't going to call a new service offering stable.

Comment (arschles) [reply docs/stability.md]: Good point - I'll make it more clear in here

**Comment by arschles** [comment]
@carolynvs I've updated to address your comments. Can you check it out again?

File: docs/stability.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+account if any of the OSBA services change in stability. Read on for how this works.
+
+# Service Stability
+
+We indicate a stability for each service in the OSBA catalog and docs to
+indicate how mature it is. Each service may have a different stability:
+
+* `experimental` - We have an idea for a new service in the catalog that we want to support,
+  but don't have a good idea how it should look. We are essentially
+  “throwing something against the wall” to see if it'll stick. Experimental
```
Comment (carolynvs) [inline docs/stability.md]: TRANSPARENCY! 😀

**Comment by krancour** [comment]
Since this is a doc-only change and nothing in our pipeline tests docs (yet), there's no point wasting resources. I've terminated the CI run to conserve them.

**Comment by arschles** [comment]
Thanks @krancour !

**Pull request merged**